backend/core/database.py

```diff
@@ -831,39 +867,39 @@ def handle_hijack_multiple_action(self, message):
 
         def _insert_bgp_updates(self):
             try:
-                cmd_ = 'INSERT INTO bgp_updates (prefix, key, origin_as, peer_asn, as_path, service, type, communities, ' \
-                    'timestamp, hijack_key, handled, matched_prefix, orig_path) VALUES ' \
-                    '%s ON CONFLICT(key, timestamp) DO NOTHING;'
-                psycopg2.extras.execute_values(
-                    self.db_cur, cmd_, self.insert_bgp_entries, page_size=1000)
-                self.db_conn.commit()
+                query = ('INSERT INTO bgp_updates (prefix, key, origin_as, peer_asn, as_path, service, type, communities, '
+                         'timestamp, hijack_key, handled, matched_prefix, orig_path) VALUES %s')
+                with get_wo_cursor(self.wo_conn) as db_cur:
```

Comment:
where is the on conflict dealt with now? (with redis primitives I mean)

Reply:
there will never be a conflict because we do a `getset` above

Reply:
I think we should never flush the redis instance or we should bootstrap values when starting